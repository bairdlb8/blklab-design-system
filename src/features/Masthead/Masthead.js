import React from 'react'
import PropTypes from 'prop-types'

import './Masthead.scss'

const Masthead = ({className=''}) => {
  return <div>Masthead</div>
}

Masthead.propTypes = {}

export default Masthead
