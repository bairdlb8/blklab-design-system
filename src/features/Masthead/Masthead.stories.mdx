import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks'

import Masthead from './Masthead'

<Meta title="Features / Masthead" component={ Masthead } />

# Masthead

## Add some text here

export const Template = (args) => {
  return <Masthead {...args} />
}

<Preview withToolbar>
  <Story name="Primary" args={ {} }>
    {Template.bind({})}
  </Story>
</Preview>

<Props of={ Masthead } />
