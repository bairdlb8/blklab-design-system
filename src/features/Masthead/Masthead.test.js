import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import { toHaveNoViolations } from 'jest-axe'
import Masthead from './Masthead'

expect.extend(toHaveNoViolations)

describe('<Masthead />', () => {

  describe('accessibility', () => {
    it('should not fail any accessibility tests', async () => {
      const { container } = render(<Masthead />)
      expect(await axe(container)).toHaveNoViolations()
    })
  })

  it('should take a snapshot', () => {
    const { asFragment } = render(<Masthead />)

    expect(asFragment(<Masthead />)).toMatchSnapshot()
  })

})
