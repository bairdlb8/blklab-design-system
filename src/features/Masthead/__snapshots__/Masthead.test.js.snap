// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Masthead /> should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    Masthead
  </div>
</DocumentFragment>
`;
